vpp/abx/vpp2202/test/test_abx.py

```diff
@@ -0,0 +1,577 @@
+#!/usr/bin/env python3
+
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2022 PANTHEON.tech
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from framework import VppTestCase, VppTestRunner
+from vpp_sub_interface import VppDot1QSubint
+from vpp_acl import AclRule, VppAcl, VppAclInterface
+
+from ipaddress import IPv4Network, IPv6Network
+
+from scapy.layers.l2 import Ether, Dot1Q
+from scapy.layers.inet import IP, TCP
+
+
+class TestABX(VppTestCase):
```

Comment:
Could you add a comment for the `TestABX` with a list of test cases we are testing here? To save time for other devs/testers needing to go through entire file.

Reply:
Adding section with tests included.